package.json

```diff
@@ -4,28 +4,27 @@
   "version": "0.13.3",
   "main": "./lib/ziti",
   "scripts": {
-    "build": "npm run build:init; npm run build:c-sdk; npm install --build-from-source --clang=1",
+    "build": "npm run build:init; npm install --build-from-source --clang=1",
     "build:init": "git submodule update --init --recursive",
-    "build:c-sdk": "cd deps/ziti-sdk-c; git submodule update --init --recursive; mkdir build; cd build; cmake -G Ninja -DCMAKE_POSITION_INDEPENDENT_CODE=ON --config Debug ..; cmake --build . --target all",
-    "build:nodejs": "node-gyp clean configure --debug build --debug -g --mmacosx-version-min=10.13",
+    "build:nodejs": "cmake-js clean configure --debug build --debug -g --mmacosx-version-min=10.13",
     "build:nodejsdbg": "npm install --build-from-source --debug -g",
-    "build:cleanup": "cd deps/ziti-sdk-c && rm -rf build",
-    "clean": "node-gyp clean; rm -rf build; cd deps/ziti-sdk-c && rm -rf build",
+    "clean": "cmake-js clean",
     "test": "echo \"Error: no test specified\"",
-    "configure": "node-gyp configure --debug",
+    "configure": "cmake-js configure",
     "start": "node main.js ./config/identity.json chapel-hill",
-    "install": "node-pre-gyp install --fallback-to-build",
+    "install:old": "node-pre-gyp install --fallback-to-build",
+    "install": "cmake-js configure && cmake-js build -- --target ziti_sdk_nodejs",
     "install-dbg": "npm install --build-from-source --clang=1 --debug",
     "publish": "node-pre-gyp package publish",
     "docs": "jsdoc2md -t API_REFERENCE.hbs lib/*.js > API_REFERENCE.md"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/netfoundry/ziti-sdk-nodejs.git"
+    "url": "https://github.com/openziti/ziti-sdk-nodejs.git"
   },
   "binary": {
     "module_name": "ziti_sdk_nodejs",
-    "module_path": "./build/{configuration}/{node_abi}-{platform}-{arch}/",
+    "module_path": "./build/{configuration}/",
     "remote_path": "./{module_name}/v{version}/{configuration}/",
```

Comment:
the gyp stuff was good at producing the magic needed to allow anyone, on any combo of node/OS/arch to automatically install the correct binaries when the user did just an `npm i`

will the cmake-js stuff do this as well?